--- a/Makefile
+++ b/Makefile
@@ -148,9 +148,6 @@ list-author-emails:
 
 
 mypy: ${PYSOURCES}
-	rm -Rf typeshed/2.7/ruamel/yaml
-	ln -s $(shell python -c 'from __future__ import print_function; import ruamel.yaml; import os.path; print(os.path.dirname(ruamel.yaml.__file__))') \
-		typeshed/2.7/ruamel/yaml
 	rm -Rf typeshed/2.7/schema_salad
 	ln -s $(shell python -c 'from __future__ import print_function; import schema_salad; import os.path; print(os.path.dirname(schema_salad.__file__))') \
 		typeshed/2.7/schema_salad
